src/components/StoreItemCard/style.less

```diff
@@ -1,15 +1,81 @@
 @import '../../styles/vars.less';
 
 .item-card {
-  border: 5px solid @gray-4;
-  display: inline-block;
   font-family: Roboto;
-  height: @item-rectangle-height;
-  margin: 15px;
-  padding: 0 30px;
+  padding: 0, 30px;
   width: @item-card-width;
-}
 
-.item-card:first-of-type {
-  margin-left: 0;
+  .inside-rectangle {
+    display: block;
+    display: flex;
+    height: @item-rectangle-height;
```

Comment:
2 Displays?

Reply:
I removed the first rule in f55372d, it might have broken looks, though. I will make sure soon.